miner/src/lib.rs

```diff
@@ -39,6 +38,12 @@ extern crate rustc_hex;
 #[cfg(test)]
 extern crate ethkey;
 
+extern crate ethabi;
```

Comment:
Move `ethabi` to other imports and `ethabi_derive/ethabi_contract` to `macro-use` imports